tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_dryrun_output.ksh

```diff
@@ -0,0 +1,142 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2020 Attila Fülöp <attila@fueloep.org>
+#
+
+. $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/cli_root/zpool_create/zpool_create.shlib
+
+typeset TMPFILE_PREFIX="$TEST_BASE_DIR/zpool_create_dryrun_output"
+typeset STR_DRYRUN="would create '$TESTPOOL' with the following layout:"
+typeset VDEV_PREFIX="$TEST_BASE_DIR/filedev"
+
+#
+# DESCRIPTION:
+# 'zpool create -n <pool> <vdev> ...' can display the correct configuration
+#
+# STRATEGY:
+# 1. Create a storage pool
+#
+
+typeset dev_size=4M
```

Comment:
4M is below the minimum vdev size.  Since this is just a dry-run nothing actually fails, but I'd still suggest bumping this to `$SPA_MINDEVSIZE`.  Since you're using `truncate` to create the file vdevs it won't actually increase the space used by the test.

Reply:
The intention was indeed to save space, but you're right since this is `truncate` it won't waste any space. I somehow missed that fact. Will change back to `$SPA_MINDEVSIZE`.